docs/docs/Concepts/embedded-chat-widget.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Embedded chat widget
+slug: /embedded-chat-widget
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ChatWidget from '@site/src/components/ChatWidget';
+
+On the [Publish pane](/concepts-publish), the **Embed into site** tab displays code that can be inserted in the `<body>` of your HTML to interact with your flow.
+
+The chat widget is implemented as a web component called `langflow-chat` and is loaded from a CDN. For more information, see the [langflow-embedded-chat repository](https://github.com/langflow-ai/langflow-embedded-chat).
+
+For a sandbox example, see the [Langflow embedded chat CodeSandbox](https://codesandbox.io/p/sandbox/langflow-embedded-chat-example-dv9zpx).
+
+This example includes the minimum required props for using the chat widget in your HTML code, which are `host_url` and `flow_id`.
```

Comment:
What example is "This example" referring to? Also, I assume that "props" = "properties", but it's not clear.